Sources/ObjCSupport/Builders/LOKOverlayLayoutBuilder.m

```diff
@@ -26,26 +26,34 @@ @interface LOKOverlayLayoutBuilder ()
 
 @implementation LOKOverlayLayoutBuilder
 
-- (instancetype)initWithPrimaryLayout:(id<LOKLayout>)primaryLayout {
+- (instancetype)initWithPrimaryLayouts:(NSArray< id<LOKLayout> > *)primaryLayouts {
     self = [super init];
-    _privatePrimary = primaryLayout;
+    _privatePrimary = primaryLayouts;
     _privateBackground = @[];
     _privateOverlay = @[];
     return self;
 }
 
+- (instancetype)initWithPrimaryLayout:(id<LOKLayout>)primaryLayout {
+    return [self initWithPrimaryLayouts:@[primaryLayout]];
+}
+
++ (instancetype)withPrimaryLayouts:(NSArray< id<LOKLayout> > *)primaryLayouts {
+    return [[self alloc] initWithPrimaryLayouts:primaryLayouts];
+}
+
 + (instancetype)withPrimaryLayout:(id<LOKLayout>)primaryLayout {
     return [[self alloc] initWithPrimaryLayout:primaryLayout];
 }
 
 - (LOKOverlayLayout *)layout {
-    return [[LOKOverlayLayout alloc] initWithPrimary:self.privatePrimary
-                                          background:self.privateBackground
-                                             overlay:self.privateOverlay
-                                           alignment:self.privateAlignment
-                                         viewReuseId:self.privateViewReuseId
-                                           viewClass:self.privateViewClass
-                                           configure:self.privateConfigure];
+    return [[LOKOverlayLayout alloc] initWithPrimaries:self.privatePrimary
+                                            background:self.privateBackground
```

Comment:
`primaries` sounds odd, so let's change these names to include the word layout for the ObjC API. `primaries` would become `primaryLayouts`, `background` would become `backgroundLayouts`, and same for overlay. Thanks.